server/handlers/policies_handlers.go

```diff
@@ -0,0 +1,106 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/gorilla/mux"
+	"github.com/layer5io/meshkit/models/meshmodel"
+	"github.com/layer5io/meshkit/models/meshmodel/core/types"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+)
+
+func (h *Handler) GetAllMeshmodelPolicy(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Please add comments to generate swagger docs for this endpoint. Refer to component_handler.go

Reply:
Done